content/news/2022-01-06-bevy-0.6/index.md

```diff
@@ -0,0 +1,1052 @@
+
+Note: directional shadows currently require more manual configuration than necessary (ex: manual configuration of the shadow projection). We will soon make this automatic and better quality over a larger range through cascaded shadow maps.
+
+### Point Light Shadows
+
+<div class="release-feature-authors">authors: @mtsr, Rob Swain (@superdump), @cart</div>
+
+Point lights can now cast "omnidirectional shadows", which can be enabled by setting `PointLight::shadows_enabled` to `true`:
+
+
+![point light](point_light.png)
+
+### Enabling and Disabling Entity Shadows
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Mesh entities can opt out of casting shadows by adding the {{rust_type(type="struct" crate="bevy_pbr" version="0.6.0" name="NotShadowCaster")}} component.
+
+```rust
+commands.entity(entity).insert(NotShadowCaster);
+```
+
+Likewise they can opt out of receiving shadows by adding the {{rust_type(type="struct" crate="bevy_pbr" version="0.6.0" name="NotShadowReceiver")}} component.
+
+```rust
+commands.entity(entity).insert(NotShadowReceiver);
+```
+
+### Spherical Area Lights
+
+<div class="release-feature-authors">authors: @Josh015</div>
+
+`PointLight` Components can now define a `radius` value, which controls the size of the sphere that emits light. A normal zero-sized "point light" has a radius of zero.
+
+![spherical area lights](spherical_area_lights.png)
+
+(Note that lights with a radius don't normally take up physical space in the world ... I added meshes to help illustrate light position and size)
+
+### Configurable Alpha Blend Modes
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Bevy's StandardMaterial now has an `alpha_mode` field, which can be set to `AlphaMode::Opaque`, `AlphaMode::Mask(f32)`, or `AlphaMode::Blend`. This field is properly set when loading GLTF scenes.
+
+![alpha blend modes](alpha_blend.png)
+
+### Clustered Forward Rendering 
+
+<div class="release-feature-authors">authors: Rob Swain (@superdump)</div>
+
+Modern scenes often have many point lights. But when rendering scenes, calculating lighting for each light, for each rendered fragment rapidly becomes prohibitively expensive as the number of lights in the scene increases. [Clustered Forward Rendering](http://www.aortiz.me/2018/12/21/CG.html) is a popular approach that increases the number of lights you can have in a scene by dividing up the view frustum into "clusters" (a grid of sub-volumes). Each cluster is then assigned lights based on whether or not they can affect that cluster. This is a form of "culling" that enables fragments to ignore lights that aren't assigned to their cluster.
+
+In practice this can significantly increase the number of lights in the scene:
+
+![clustered forward rendering](clustered_forward_rendering.png)
+
+Clusters are "3d" slices of the view frustum, but when debugging them in screen space they look like squares (which extend "in" to the screen):
```

Comment:
Maybe this is too complicated, but I don't like "3d slices". I'm trying to communicate the idea of them being sub-frusta in a simple way without being incorrect.
```suggestion
Clusters are 3d subdivisions of the view frustum. They are cuboids in projected space so for a perspective projection, they are stretched and skewed in view space. When debugging them in screen space, you are looking along a row of clusters and so they look like squares. Different colors within a square represent mesh surfaces being at different depths in the scene and so they belong to different clusters:
```